docs/Authenticating-Users-via-Tokens.md

```diff
@@ -60,7 +66,43 @@ The following properties must be present in portal.properties in order to allow
 - **Permissible Values**: true or false
 - **Default Value**: true
 
+**Property**: dat.oauth2.clientId (required only when dat.method = oauth2)
+- **Description**: Identifier of the OAuth2 client of the authentication provider.
+- **Permissible Values**: string
+- **Default Value**: This value is specified by the authentication provider.
+
+**Property**: dat.oauth2.clientSecret (required only when dat.method = oauth2)
+- **Description**: Secret of the OAuth2 client of the authentication provider used for back channel communication between cBioPortal and the provider. This value is specified by the authentication provider.
+- **Permissible Values**: string
+- **Default Value**: This value is specified by the authentication provider.
+
+**Property**: dat.oauth2.issuer (required only when dat.method = oauth2)
+- **Description**: Name of the issuer of the tokens. It is used to assert the correct origin of the token. This value is specified by the authentication provider.
+- **Permissible Values**: string
+- **Default Value**: This value is specified by the authentication provider.
+
+**Property**: dat.oauth2.userAuthorizationUri (required only when dat.method = oauth2)
+- **Description**: Url of the user login page of the OAuth2 authorization provider. According to OAuth2 specification this endpoint must end with `/auth`.
+- **Permissible Values**: url
+- **Default Value**: This value is specified by the authentication provider.
+
+**Property**: dat.oauth2.accessTokenUri (required only when dat.method = oauth2)
+- **Description**: Url for retrieval of tokens from the OAuth2 authorization provider. According to OAuth2 specification this endpoint must end with `/token`.
+- **Permissible Values**: url
+- **Default Value**: This value is specified by the authentication provider.
+
+**Property**: dat.oauth2.jwkUrl (required only when dat.method = oauth2)
+- **Description**: Url for retrieval of the public key used by the OAuth2 authorization provider to sign tokens. According to OAuth2 specification this endpoint must end with `/certs`.
+- **Permissible Values**: url
+- **Default Value**: This value is specified by the authentication provider.
+
+**Property**: dat.oauth2.jwtRolesPath (required only when dat.method = oauth2)
+- **Description**: `::`-separated path to array with user roles in JWT token returned by Keycloak.
+- **Permissible Values**: `::`-separated strings
+- **Default Value**: resource_access::cbioportal::roles
+
```

Comment:
I think you are missing dat.oauth2.redirectUri

Reply:
Fixed